### DISCUSSION TITLE
Add a tool that allows you to set what a new player will spawn with when they open your world

### DISCUSSION THREAD

**Discussion by G01dY05h1** [Mojang/minecraft-editor#78] [Feedback]
Let's say a player were to download a world I have made, when they open it, they will spawn with items pre-selected by me, they only receive the items one time and other players also receive them upon joining.

**Comment by CaptainCooky** [comment]
This can already be accomplished with command blocks. 😁

This here link, be a good place to start! 👍

https://learn.microsoft.com/en-us/minecraft/creator/documents/commandsintroduction

**Reply by G01dY05h1** [reply]
While I had thought of this while making the discussion, I thought it would be a lot easier if the ability to do this was built in. Since this is likely something that a person making a map would want to do, as well as the fact a built in GUI for this functionality would make it as easy as grabbing items from the creative menu as if you were giving yourself the items.

This also gives me an idea that you could save and load a state to test your world similar to Mario Maker. I will make a separate discussion for that if one does not already exist. (A similar discussion already exists [here](https://github.com/Mojang/minecraft-editor/discussions/64))
